inspire/utils/bibtex_booktitle.py

```diff
@@ -43,8 +43,8 @@ def generate_booktitle(record):
                         if acronym:
                             booktitle = "%s: %s" % (rn, acronym, )
                         else:
-                            recids = search_pattern(p="reportnumber:%s"
-                                                    % (rn,))
+                            recids = Query(p="reportnumber:%s" % (rn,))\
```

Comment:
I was getting back a search results object , I will double check to be sure.

2015-08-25 14:46 GMT+02:00 Javier Martin Montull <notifications@github.com>:

> In inspire/utils/bibtex_booktitle.py
> <https://github.com/inspirehep/inspire-next/pull/318#discussion_r37860202>
> :
>
> > @@ -43,8 +43,8 @@ def generate_booktitle(record):
> >                          if acronym:
> >                              booktitle = "%s: %s" % (rn, acronym, )
> >                          else:
> > -                            recids = search_pattern(p="reportnumber:%s"
> > -                                                    % (rn,))
> > +                            recids = Query(p="reportnumber:%s" % (rn,))\
>
> I think if you want the same behaviour as before this should be:
>
> Query(p="reportnumber:%s" % (rn,)).search().recids
>
> In order to get record ids back. Otherwise you might receive back the full
> json by default. Did you check?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/318/files#r37860202>.
>